packages/host/tests/integration/components/schema-test.gts

```diff
@@ -179,7 +178,8 @@ module('Integration | schema', function (hooks) {
       );
   });
 
-  test('renders link to field card', async function (assert) {
+  //  Temporarily disable this until we determine architectural decisions for code mode
+  skip('renders link to field card', async function (assert) {
```

Comment:
let's just delete. if this ever does come back i think it will be in a very different form than we have now